components/locid/Cargo.toml

```diff
@@ -46,6 +46,7 @@ icu = { path = "../icu", default-features = false }
 icu_benchmark_macros = { version = "0.5", path = "../../tools/benchmark/macros" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+postcard = { version = "0.7.0", features = ["use-std"] }
```

Comment:
Nit: Apply `default-features = false` whenever we link `postcard` because we want to avoid the `heapless-cas` feature being pulled in because it contains undesirable dependencies. It's less of a problem for `dev-dependencies` but still good practice.

Reply:
Done